detox/android/detox/src/full/java/com/wix/detox/reactnative/idlingresources/DescriptiveIdlingResource.kt

```diff
@@ -0,0 +1,5 @@
+package com.wix.detox.reactnative.idlingresources
+
+interface DescriptiveIdlingResource {
```

Comment:
Just wondering if it makes sense to extend the `IdlingResource` in order not to write that twice, e.g.:

```diff
-public abstract class DetoxBaseIdlingResource implements IdlingResource, DescriptiveIdlingResource {
+public abstract class DetoxBaseIdlingResource implements DescriptiveIdlingResource {
```

Reply:
Good thinking. Really, it's not a pluggable that adds a methods to whatever, rather a very specific addition to idling-resources 👍🏻